src/test/java/seedu/address/model/studyspot/RatingTest.java

```diff
@@ -39,4 +41,12 @@ public void isValidRating() {
         assertTrue(Rating.isValidRating("5"));
         assertTrue(Rating.isValidRating("1"));
     }
+
+    @Test
+    public void hashCode_test() {
+        Rating rating = new Rating("2");
+
+        assertEquals(new Rating("2").hashCode(), rating.hashCode());
+        assertNotEquals(new Rating("5").hashCode(), rating.hashCode());
+    }
 }
```

Comment:
Missing equals test when this != other

Reply:
Not sure what you mean?

I'm testing for `Rating(2)_B == Rating(2)_A`
and `Rating(5) != Rating(2)_A`